packages/amplify-e2e-core/src/utils/pinpoint.ts

```diff
@@ -30,6 +30,7 @@ const serviceRegionMap = {
   'us-west-2': 'us-west-2',
   'cn-north-1': 'us-west-2',
   'cn-northwest-1': 'us-west-2',
+  'ap-east-1': 'ap-northeast-2',
```

Comment:
Pinpoint is not available in ap-east-1, so falling back to ap-northeast-2 

https://docs.aws.amazon.com/general/latest/gr/pinpoint.html

Reply:
Could you also add this comment in the code? So that it will be much helpful once this part is revisited